src/wolframite/base/parse.clj

```diff
@@ -49,9 +50,9 @@
 
 ;; FIXME: change name (it's more of a concrete type map)
 (defn bound-map [f coll {:keys [flags] :as opts}]
-  (if (options/flag?' flags :vectors)
-    (mapv #(f % opts) coll)
-    (map  #(f % opts) coll)))
+  (if (options/flag?' flags :seqs)

```

Comment:
switched the flag and order so that :vectors is truly the default, even without having to specify it.